main.tf

```diff
@@ -237,6 +235,25 @@ resource "null_resource" "run_command" {
     EOT
   }
 
+}
+
+resource "null_resource" "run_destroy_command" {
```

Comment:
What happens with this the first time you run? Anything?

Reply:
The resource gets created. It shows up in the plan like this:
```
# module.agent-policy.module.gcloud.null_resource.run_destroy_command[0] will be created
  + resource "null_resource" "run_destroy_command" {
      + id       = (known after apply)
      + triggers = {
          + "destroy_cmd_body"       = "<my-project-id> \"<my-policy-id>\""
          + "destroy_cmd_entrypoint" = "agent-policy/scripts/delete-script.sh"
          + "gcloud_bin_abs_path"    = "/google-cloud-sdk/bin"
        }
    }
```

Reply:
Ok great, sounds like it does exactly what we want. 👍

Reply:
Nice!